### DISCUSSION TITLE
Vote for zzl200012 as RisingLight member

### DISCUSSION THREAD

**Discussion by skyzh** [risinglightdb/risinglight#439] [Member]
@zzl200012 has made great contributions to the RisingLight's storage engine in the past month. Here's a list of their pull requests: https://github.com/risinglightdb/risinglight/pulls?q=author%3Azzl200012

In https://github.com/risinglightdb/risinglight/pull/414, @zzl200012 improved how our system handles data ingestion. By encoding the RowSet while doing inserts, this patch makes ingestion faster.

In https://github.com/risinglightdb/risinglight/pull/424, @zzl200012 greatly improved the filter scan functionality. They combined the delete vector processing with the filter scan visibility map, which reduces duplicated code as well as boost efficiency in large deletions.

@zzl200012 demonstrated a good understanding of the Rust programming language and the RisingLight project. Therefore, I'd like to promote them as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. A minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by stephenkgli** [comment]
Take me away!

**Comment by skyzh** [comment]
With 4 member votes from @skyzh @wangrunji0408 @likg227 @st1page, we are accepting this vote. Welcome to the RisingLight community!

**Reply by skyzh** [reply]
... the invitation email should be sent. And @zzl200012 will be granted write access to all our repos.
